eq-author/src/App/QuestionnairesPage/QuestionnairesView/index.test.js

```diff
@@ -811,5 +822,61 @@ describe("QuestionnairesView", () => {
         expect(getByText("1 of 2")).toBeTruthy();
       });
     });
+    describe("Filtering", () => {
+      it("should filter unowned questionnaires out when Owner mode selected", () => {
+        props.questionnaires = [
+          buildQuestionnaire(1, {
+            createdBy: {
+              id: "3",
+              name: "test user",
+              email: "test@email.com",
+              displayName: "test user",
+            },
+          }),
+          buildQuestionnaire(2, {
+            createdBy: {
+              id: "2",
+              name: "test user",
+              email: "test@email.com",
+              displayName: "test user",
+            },
+          }),
+          buildQuestionnaire(3, {
+            createdBy: {
+              id: "1",
+              name: "test user",
+              email: "test@email.com",
+              displayName: "test user",
+            },
+          }),
+        ];
+
+        const { getByLabelText, queryByText } = render(
+          <QuestionnairesView
+            {...props}
+            questionnaires={props.questionnaires}
+          />
+        );
+
+        const showAllButton = getByLabelText("All");
+        const hideUnownedButton = getByLabelText("Owner");
+
+        expect(queryByText("Questionnaire 1 Title")).toBeTruthy();
+        expect(queryByText("Questionnaire 2 Title")).toBeFalsy();
+        expect(queryByText("Questionnaire 3 Title")).toBeFalsy();
+
+        fireEvent.click(showAllButton);
+
+        expect(queryByText("Questionnaire 1 Title")).toBeTruthy();
+        expect(queryByText("Questionnaire 2 Title")).toBeTruthy();
+        expect(queryByText("Questionnaire 3 Title")).toBeTruthy();
+
+        fireEvent.click(hideUnownedButton);
```

Comment:
Could this be `showOwnedButton`? I find this double negative (hide and un) a bit confusing